packages/grafana-ui/src/components/uPlot/geometries/configGetters.ts

```diff
@@ -1,10 +1,10 @@
 import { AreaProps, LineProps, PointProps } from './types';
 import tinycolor from 'tinycolor2';
-import { getColorFromHexRgbOrName } from '@grafana/data';
 
 export const getAreaConfig = (props: AreaProps) => {
+  // TODO can we pass therem here? or make sure color is already correct?
   const fill = props.fill
-    ? tinycolor(getColorFromHexRgbOrName(props.color))
```

Comment:
@dprokop found this part in the new graph component, but do not have access to the theme here so can't get the correct color

Reply:
yes, great catch!